README.md

```diff
@@ -108,11 +109,14 @@ Build package
 ./gradlew clean test shadowJar
 ```
 
-Setup environment variables
+Setup local bucket
 
 ```bash
-export ECS_ACCOUNT_ID=dev
-export AWS_PROFILE=befaster    # pre-configured profile contained in ~/.aws/credentials
+export AWS_PROFILE=befaster                      # pre-configured profile contained in ~/.aws/credentials
+
+minio mb myminio
+minio mb myminio/tdl-test-auth/TCH/user01/
+minio cp ./build/resources/test/HmmmLang_R1Cov33_R2Cov44.srcs myminio/tdl-test-auth/TCH/user01/test1.srcs
```

Comment:
This is what the acceptance test is doing.
We do not need to do it manually.

Reply:
That's good when we run the acceptance test, although I think these manual steps are good to have if we want to run the lambda manually and not via the test.

Reply:
So both the acceptance test and manual run of the lambda is useful in my view, otherwise just those single serveless commands in the README do not serve a purpose.